### PR TITLE
Add versionist with the default configuration

### PR DESCRIPTION
**Pull request by pcarranzav**
We also add the possibility to use `npm run versionist` so that we
can use the local versionist version installed from package.json.
Also rewrite the changelog to match the versionist default format.

Change-Type: patch
Signed-off-by: Pablo Carranza Velez <pablo@resin.io>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,27 +1,33 @@
...
+This project adheres to [Semantic Versioning](http://semver.org/).
 
-# 1.3.1
+# 1.3.2 - 2016-11-18
 
-* Fixed promisifying docker.runAsync so it returns all callback results.
+* Fix: promisify all Docker functions [Pablo Carranza Velez]
 
-# 1.3.0
+# 1.3.1 - 20160-11-09
```
Comment (lurch) [inline CHANGELOG.md]: Date typo ;)

Comment (pcarranzav) [inline CHANGELOG.md]: It was actually a version from the future, but thanks for reminding me we have to keep that kind of things secret.

File: versionist.conf.js [hunk 2]
```diff
@@ -0,0 +1 @@
+module.exports = require('versionist/versionist.conf');
```
Comment (lurch) [inline versionist.conf.js]: Would https://github.com/resin-io/versionist/issues/58 make this file unnecessary?

Comment (pcarranzav) [inline versionist.conf.js]: Ah, yes, I hadn't seen that one. I'd leave this here anyways until that issue is resolved.

**Pull request merged**